### PR TITLE
Styles Grouping/Sorting #1770

### PR DESCRIPTION
**Pull request by hswlab**
As described in https://github.com/lllyasviel/Fooocus/issues/1770#issuecomment-1887719446

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -218,3 +218,49 @@
...
+                  /* max-width: calc(35% - 15px) !important; */ /* add this to enable 3 columns layout */
+                  flex: calc(50% - 5px) !important;
+                  padding: 2px 5px;
+              }
+
+                  .contain .tabs .tabitem.style_selections_tab .style_selections .wrap[data-testid="checkbox-group"] label span {
+                      /* white-space:nowrap; */ /* add this to disable text wrapping (better choice for 3 columns layout) */
+                      overflow: hidden;
+                      text-overflow: ellipsis;
+                  }
```
Comment (mashb1t) [inline css/style.css:257]: Please adjust the style indent here:

```suggestion
.contain .tabs .tabitem.style_selections_tab {
  height: 100%;
}

.contain .tabs .tabitem.style_selections_tab > div:first-child {
  height: 100%;
}

.contain .tabs .tabitem.style_selections_tab .style_selections {
  min-height: 200px;
  height: 100%;
}

.contain .tabs .tabitem.style_selections_tab .style_selections .wrap[data-testid="checkbox-group"] {
  position: absolute; /* remove this to disable scrolling within the checkbox-group */
  overflow: auto;
  padding-right: 2px;
  max-height: 100%;
}

.contain .tabs .tabitem.style_selections_tab .style_selections .wrap[data-testid="checkbox-group"] label {
  /* max-width: calc(35% - 15px) !important; */ /* add this to enable 3 columns layout */
  flex: calc(50% - 5px) !important;
  padding: 2px 5px;
}

.contain .tabs .tabitem.style_selections_tab .style_selections .wrap[data-testid="checkbox-group"] label span {
  /* white-space:nowrap; */ /* add this to disable text wrapping (better choice for 3 columns layout) */
  overflow: hidden;
  text-overflow: ellipsis;
}
```

File: javascript/script.js [hunk 2]
```diff
@@ -162,24 +165,28 @@ function initStylePreviewOverlay() {
...
-        if(!overlayVisible) return;
+    document.addEventListener('mousemove', function (e) {
+        if (!overlayVisible) return;
         overlay.style.left = `${e.clientX}px`;
         overlay.style.top = `${e.clientY}px`;
         overlay.className = e.clientY > window.innerHeight / 2 ? "lower-half" : "upper-half";
     });
 }
 
+
```
Comment (mashb1t) [inline javascript/script.js:189]:
```suggestion
```

**Review by mashb1t — CHANGES_REQUESTED** [review]
LGTM, just small code style changes

**Comment by hswlab** [comment]
It's my first PR with review at github, I hope I didn't miss to click something ^^'.

**Review by mashb1t — APPROVED** [review]

**Pull request merged**